examples/bevy/README.md

```diff
@@ -1,12 +1,42 @@
 # Bevy Game Engine Example
+This code demonstrates a minimal example of how to use the [bevy] game engine inside the RISC Zero [zkVM].
 
-This code demonstrates a minimal example of how code from the bevy game engine can be used.
+## Quick Start
+First, [install Rust] if you don't already have it.
+Then, run the example with:
+```
+  cargo run --release
+```
 
-## Proving movement
+## Use Cases
+By using this demo as part of a [Bonsai application], you could build an app where on-chain payment depends on off-chain gameplay.
+To learn more about this use case, check out our Twitter thread about using Bonsai as a [zk coprocessor].
 
-This example takes as input an integer (call it `n`). When the program is run, a
-world and a primary entity are created. After creation, the primary entity is
-moved `n` units on the x-axis and writes the position in the journal. The host
-can verify the receipt and also determine that the primary entity moved `n`
-units. While this is a fairly minimal example, we encourage you to bring your
-creativity to take this example to the next level!
+To link gameplay to a particular player, you may want to pair this demo with the [ECDSA] demo, which would allow a player to sign their moves.
+
+## Project Organization
+zkVM applications are organized into a [host program] and a [guest program].
+The host program can be found in [`src/main.rs`](src/main.rs) and the guest program can be found in [`methods/guest/src/main.rs`](methods/guest/src/main.rs).
+
+The [host] first [executes] the guest program and then [proves the execution] to construct a [receipt].
```

Comment:
The `[proves the execution]` link is dead -- either switch it to point to `#prover` not `#prove`, or add a `#prove` section to the terminology page. Personally I lean toward the latter.